qiskit/qasm3/ast.py

```diff
@@ -136,6 +136,7 @@ class ClassicalType(ASTNode):
 class FloatType(ClassicalType, enum.Enum):
     """Allowed values for the width of floating-point types."""
 
+    UNSPECIFIED = 0
```

Comment:
Similar to the type story - let's just be explicit and always have Qiskit output double precision for now.  (Fwiw, in a sane AST - which this is absolutely not an example of lol -, I think we would have called this enum `FloatWidth`, and had `FloatType` be a record type with an optional `FloatWidth`.)